LocalSettings.php

```diff
@@ -4644,6 +4644,11 @@
  				'unreviewedpages' => true,
  			),
  		),
+		'+scruffywiki' => array(
+			'sysop' => array(
```

Comment:
Fix the indenting please.

Reply:
I can only see tabs in the diff?

Reply:
Take a look at how it’s done on other wikis. See how the individual group arrays are indented one tab more then the wiki array? Here instead of that you just indented it one space.